bin/templates/cordova/lib/builders/ProjectBuilder.js

```diff
@@ -295,12 +294,18 @@ class ProjectBuilder {
     }
 
     findOutputApks (build_type, arch) {
-        var self = this;
-        return Object.keys(this.binDirs).reduce(function (result, builderName) {
-            var binDir = self.binDirs[builderName];
-            return result.concat(findOutputApksHelper(binDir, build_type, builderName === 'ant' ? null : arch));
+        return Object.keys(this.binDirs).reduce((result, builderName) => {
```

Comment:
couldn't `binDirs` and this code be simplified to just a string?

Reply:
Yes, I also think this can be simplified. 

Originally I was going to but decided to wait and clean up the other other areas first. Thanks for reminding me this one. I will take a look at it again.